app/services/message_sender.rb

```diff
@@ -15,12 +16,19 @@ def run
         client.chat_postMessage(
           channel: channel_id,
           as_user: true,
-          text: message.body
+          text: escaped_message_body
         )
       end
     end
   end
 
+  def escape_specific_characters(message_body)
```

Comment:
Are you thinking of a small utils-like class for the Message/MessageSender object(s) to deal with any content formatting or manipulation instead?  This way if we have to add other things of this nature in the future they have a dedicated place to live?